tests/unit/uncertainty/test_evaluators.py

```diff
@@ -1,18 +1,157 @@
-# import pytest
-# import torch
-
-# from chemprop.uncertainty.evaluator import (
-#     CalibrationAreaEvaluator,
-#     ConformalMulticlassEvaluator,
-#     ConformalMultilabelEvaluator,
-#     ConformalRegressionEvaluator,
-#     ExpectedNormalizedErrorEvaluator,
-#     MetricEvaluator,
-#     NLLClassEvaluator,
-#     NLLMultiEvaluator,
-#     NLLRegressionEvaluator,
-#     SpearmanEvaluator,
-# )
+import pytest
+import torch
+
+from chemprop.uncertainty.evaluator import (  # CalibrationAreaEvaluator,; ConformalMulticlassEvaluator,; ConformalMultilabelEvaluator,; ConformalRegressionEvaluator,; ExpectedNormalizedErrorEvaluator,; MetricEvaluator,
+    NLLClassEvaluator,
+    NLLMultiEvaluator,
+    NLLRegressionEvaluator,
+    SpearmanEvaluator,
+)
+
+
+@pytest.mark.parametrize(
+    "preds,uncs,targets,mask,likelihood",
+    [
+        (
+            torch.tensor([[0.8]]),
+            torch.tensor([[0.8]]),
+            torch.ones([1, 1]),
+            torch.ones([1, 1], dtype=bool),
+            torch.tensor([0.8]),
+        ),
+        (
+            torch.tensor([[0.8]]),
+            torch.tensor([[0.8]]),
+            torch.zeros([1, 1]),
+            torch.ones([1, 1], dtype=bool),
+            torch.tensor([0.2]),
+        ),
+    ],
+)
+def test_NLLClassEvaluator(preds, uncs, targets, mask, likelihood):
+    """
+    Testing the NLLClassEvaluator
+    """
+    evaluator = NLLClassEvaluator()
+    nll_calc = evaluator.evaluate(preds, uncs, targets, mask)
+    likelihood_calc = torch.exp(-1 * nll_calc)
+    torch.testing.assert_close(likelihood_calc, likelihood)
+
+
+@pytest.mark.parametrize(
+    "preds,uncs,targets,mask,likelihood",
+    [
+        (
+            torch.tensor(
+                [
+                    [[0.29, 0.22, 0.49]],
+                    [[0.35, 0.19, 0.46]],
+                    [[0.55, 0.38, 0.07]],
+                    [[0.15, 0.29, 0.56]],
+                    [[0.08, 0.68, 0.24]],
+                ]
+            ),
+            torch.tensor(
+                [
+                    [[0.29, 0.22, 0.49]],
+                    [[0.35, 0.19, 0.46]],
+                    [[0.55, 0.38, 0.07]],
+                    [[0.15, 0.29, 0.56]],
+                    [[0.08, 0.68, 0.24]],
+                ]
+            ),
+            torch.tensor([[0], [2], [2], [0], [1]]),
+            torch.ones([5, 1], dtype=bool),
+            torch.tensor([0.24875443]),
+        ),
+        (
+            torch.tensor(
+                [
```

Comment:
same here

Reply:
I believe it is actually the previous lines, but I have changed it to `None`.